security/tests/integration/cmd/main.go

```diff
@@ -0,0 +1,276 @@
+// Copyright 2017 Istio Authors
```

Comment:
Ideally we should use go test for this instead a go binary. Just like sample tests.